doc/source/composites.rst

```diff
@@ -107,7 +107,7 @@ being visualized.
 
 .. note::
```

Comment:
Would it be better to use the deprecated directive:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated

Reply:
Thanks; I was not aware of that one.  Fixed in 24cdc3816.